homeassistant/components/sensor/postnl.py

```diff
@@ -50,18 +52,36 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
         _LOGGER.exception("Can't connect to the PostNL webservice")
         return
 
-    add_devices([PostNLSensor(api, name)], True)
+    sensor_items = []
+    sensor_items.extend([PostNLSensor(
+        hass,
+        'PostNL-package',
+        'package-variant-closed',
+        'packeges',
+        api)])
+
+    if letter:
+        sensor_items.extend([PostNLSensor(
+            hass,
+            'PostNL-letter',
+            'email-outline',
+            'letters',
+            api)])
+
+    add_devices(sensor_items)
 
 
 class PostNLSensor(Entity):
     """Representation of a PostNL sensor."""
 
-    def __init__(self, api, name):
+    def __init__(self, hass, name, icon, unit_of_measurement, api):
         """Initialize the PostNL sensor."""
         self._name = name
         self._attributes = None
         self._state = None
         self._api = api
+        self._icon = "mdi:" + icon
```

Comment:
Please use string formatting -> `'mdi:{}'.format(icon)`

Or simpler: Just pass the icon name to the constructor, e.g., `mdi:email-outline` and `mdi:package-variant-closed`.